### PR TITLE
Misc balance and additions

### PR DESCRIPTION
**Pull request by WALLOFJUSTICE**

### CODE REVIEW

File: src/game.cpp [hunk 1]
```diff
@@ -2270,11 +2270,20 @@ int main(int argc, char** argv)
...
-			                         "Barony has encountered a critical error and cannot start.\n\n"
-			                         "Please check the log.txt file in the game directory for additional info,\n"
-			                         "or contact us through our website at http://www.baronygame.com/ for support.",
-			                         screen);
+									"Barony has encountered a critical error and cannot start.\n\n"
+									"Please check the log.txt file in the game directory for additional info\n"
+									"and verify Steam is running. Alternatively, contact us through our website\n"
+									"at http://www.baronygame.com/ for support.",
+				screen);
+#elif
```
Comment (addictgamer) [inline src/game.cpp]: Should be `#else`?

Comment (WALLOFJUSTICE) [reply src/game.cpp]: Addressed in 51b5ccb

File: src/actgeneral.cpp [hunk 2]
```diff
@@ -471,4 +471,8 @@ void actFloorDecoration(Entity* my)
 	{
 		return;
 	}
+	if ( !my->flags[PASSABLE] )
+	{
+		my->flags[PASSABLE] == true;
```
Comment (addictgamer) [inline src/actgeneral.cpp]: This code is incorrect

Comment (addictgamer) [reply src/actgeneral.cpp]: This may also be why pathfinding was breaking with grass

**Comment by addictgamer** [comment]
Merged.

**Pull request closed without merging**